pallets/funding/src/storage_migrations.rs

```diff
@@ -4,278 +4,3 @@ use frame_support::traits::StorageVersion;
 /// The current storage version
 pub const STORAGE_VERSION: StorageVersion = StorageVersion::new(3);
 pub const LOG: &str = "runtime::funding::migration";
-
-pub mod v2 {
```

Comment:
They throw out compilation errors. If we want to keep them, we would have to re-declare all inner types which would become a huge file.
I don't think it benefits having these migrations around, since we can always retrieve them on the git history.

Also I checked some examples some time ago and some chains delete them as well.

What do you think is the benefit of keeping them around?